src/Backend/MessageManagerBackendDispatcher.php

```diff
@@ -97,7 +97,7 @@ public function handle(MessageInterface $message, EventDispatcherInterface $disp
      */
     public function getStatus()
     {
-        return new Success('Channel is running (RabbitMQ) and consumers for all queues available.');
+        return new Success('Channel is running (Database) and consumers for all queues available.');
```

Comment:
what if someone is testing agents this message?

Reply:
It will break indeed, but it was a wrong one

Reply:
what do others think? cc @sonata-project/contributors

Reply:
If it was wrong, Imo lets change it

Reply:
good enough for me, let's merge it then.

Reply:
@kunicmarko20 everything is a BC-break, see my favorite xkcd ;)